### PR TITLE
Vue 2.0 support

### PR DESCRIPTION
**Pull request by aphofstede**

### CODE REVIEW

**Comment by aambrozkiewicz** [comment]
this seems very nice, what stops it from being merged?

**Comment by BrockReece** [comment]
Hi,

Sorry for the slow reply, github didn't notify me about this PR.
I am happy to merge in these changes but don't want to break anything for anyone still using Vue 1. 

Do you think I should merge these changes into a new branch and then the users can choose the version they need?

Cheers
Brock

**Comment by aphofstede** [comment]
I would tag & branch the current master as the last 1.x and continue on master with 2.0. Anyone depending directly on a head should expect breaking changes right?

## 

Alexander Hofstede

File: src/Dropdown.vue [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 <template>
 
-<div id="{{element_id}}" class="ui fluid search selection dropdown">
-    <input type="hidden" name="{{name}}" value="{{selection}}">
+<div v-bind:id="element_id" class="ui fluid search selection dropdown">
+    <input type="hidden" v-bind:name="name" value="{{selection}}">
```
Comment (OmgImAlexis) [inline src/Dropdown.vue:13]: This should be `v-bind:value="selection"`

**Review by OmgImAlexis — CHANGES_REQUESTED** [review]

**Comment by ricardovf** [comment]
@BrockReece wont this be merged?

You are working here right? https://github.com/CroudSupport/vue-semantic/commits/vue2

**Comment by BrockReece** [comment]
@ricardovf 

I found a few issues with this PR when trying it in anger in my own applications so I am using this as a starting point and fixing the little niggles as I go, I should have this up by the end of the week.

Sorry for the delay again
Brock

**Pull request closed without merging**